base/doc/ltnews42.tex

```diff
@@ -148,6 +148,39 @@ \section{Introduction}
 
 \section{News from the Tagged PDF project}
 
+\subsection{Expanding the \cs{DocumentMetadata} command}
+ 
+In 2022 we introduced the \cs{DocumentMetadata} with a twofold purpose:
+It provides a dedicated place for document wide settings and metadata. And it acts as a trigger command to identify documents that want to load new code
+and to use new extended interfaces which are needed for the Tagging Project but are useful also without tagging.
+ 
+At first using \cs{DocumentMetadata} with an empty argument loaded only the PDF management and a new hyperref driver was used. Since november 2024 \cs{DocumentMetadata} changes the default encoding from OT1 but T1 and since june 2025 it changes the default PDF version from 1.7 to 2.0.
+
+Additional code in latex-lab, needed, e.g., for the tagging project, had to be loaded explicitly by using the \texttt{testphase} or the new \texttt{tagging} key in the argument of \cs{DocumentMetadata}. While this allowed to selectively load and test the new code it also makes it difficult for classes and packages to adapt their code for
+the tagging project as it was difficult to test which parts of the latex-lab code were active.
+ 
+In this release we therefore extend \cs{DocumentMetadata} even more: it will now load directly all the code that one get also when using the \texttt{tagging=off} or the \texttt{testphase=latest} key.  
+
+The values \texttt{phase-I}, \texttt{phase-II}, \texttt{phase=III} of the \texttt{testphase} key will no longer load different code variants but only activate tagging. Extra modules not yes incorporated in the \texttt{latest} set of modules can still be loaded with the \texttt{testphase} key.
+ 
+For documents that want to load the PDF management but do not want the new tagging support code we provide a dedicated package. Such documents should replace
+\begin{verbatim}
+\DocumentMetadata{pdfversion=1.7,pdfstandard=a-3b}
+\end{verbatim}
+by
+\begin{verbatim}
+\RequirePackage{pdfmanagement}
+\SetKeys{document/metadata}{pdfversion=1.7,pdfstandard=a-3b}
+\end{verbatim}
+
+ 
+\subsection{Requiring or testing for the Tagging support code}
+Classes or packages that are written only for the new code loaded by
+\cs{DocumentMetadata} can use the new command \cs{NeedsDocumentMetadata} at the begin of the class or package file. It will error if the tagging support code has not be loaded.
+ 
+Classes and package that want to support both legacy documents and newer document using \cs{DocumentMetadata} can now use \cs{IfDocumentMetadataTF} to test if the new code has been loaded -- eventually in combination with a test of the date of the format. To test if the PDF management has been loaded the test \cs{IfPDFManagementActiveTF} is provided.
```

Comment:
document  ==>  documents 

And:
To test if the ... 
==>
To test whether the ... loaded, the ...